### ISSUE TITLE
Unable to delete values from fields

### ISSUE THREAD

**Issue by Gr8z** [final-form/react-final-form-arrays#114]
Hi,

I am trying to implement arrays on my form. Adding and initializing fields work great. But I get an error when I try to remove a value from my array.

```cmd
Uncaught TypeError: Cannot read property 'validators' of undefined
    at final-form.es.js:1074
```
Code: 
```js
onClick={() => fields.remove(index)}
```

Any pointers or guidance is appreciated.

**Comment by mikefowler** [comment]
@erikras any top-of-mind ideas about the source of this bug? My sense is that there's a binding issue somewhere. No code in this project references `.validators`, so something in this project is calling up to the core `final-form` code and throwing the error described above.

**Comment by mikefowler** [comment]
@Gr8z I figured out what was going on for me, and I'd guess it's the same for you. This was a bug in `final-form`, fixed [here](https://github.com/final-form/final-form/commit/7abee7cea09b09dd4d2a886e46c05d98f04ac568#diff-1fbdeac5e3837fb021ccc625fb2d8442). The patch was released in [`v4.18.1`](https://github.com/final-form/final-form/releases/tag/v4.18.1).

**Comment by Gr8z** [comment]
Yes, that was totally it. Thank you!

I guess this issue is resolved then.
